README.md

```diff
@@ -6,29 +6,27 @@
 
 @hyperapp/router provides utilities for routing client-side pages with [Hyperapp](https://github.com/hyperapp/hyperapp) using the [History API](https://developer.mozilla.org/en-US/docs/Web/API/History).
 
+
+Use the `router` as Hyperapp module. Then compose your routes with components (`Router`, `Route`).
+
+
 [Try it Online](http://hyperapp-router.surge.sh)
 
 ```jsx
-import { router, Link } from "@hyperapp/router"
+import { h, app } from 'hyperapp';
+import { router, Router, Route } from '@hyperapp/router';
 
 app({
-  view: [
-    [
-      "/",
-      (state, actions) =>
-        <Link to="/test" go={actions.router.go}>
-          Test
-        </Link>
-    ],
-    [
-      "/test",
-      (state, actions) =>
-        <Link to="/" go={actions.router.go}>
-          Back
-        </Link>
-    ]
-  ],
-  mixins: [router()]
+  state: { ... },
+  actions: { ... },
+  view: (state, actions) => (
+    <Router state={state} actions={actions}>
```

Comment:
@speier So, it's decided we are going to share both state and actions with each component/view? 🤔👍

Reply:
It's either:
A:
```jsx
MyComponent = props => <div>{props.foo}</div>

app({
  modules: {router},
  state: {foo: 'bar'},
  view: (state, actions) => (
    <Router state={state} actions={actions}>
      <Route path="/foo" component={_ => <MyComponent foo={state.foo} />} />
    </Router>
  )
})
```

B:

```jsx
MyComponent = props => <div>{props.state.foo}</div>

app({
  modules: {router},
  state: {foo: 'bar'},
  view: (state, actions) => (
    <Router state={state} actions={actions}>
      <Route path="/foo" component=MyComponent />
    </Router>
  )
})
```

or C:


```jsx
MyComponent = (state, actions) => <div>{state.foo}</div>

app({
  modules: {router},
  state: {foo: 'bar'},
  view: (state, actions) => (
    <Router state={state} actions={actions}>
      <Route path="/foo" component=MyComponent />
    </Router>
  )
})
```

I prefer option *B*

Reply:
@speier @zaceno I think I get it now. These are not components, React calls it components, since that's the terminology they have, but what we are dealing with here are just views, just like the old router used to route views, not components. 

I think the way to move forward here is to just rename `component` to `view` and go with what Kalman made.

Reply:
...and *D* is just *B* with a better name. So I'm good with that 😄 👍

Reply:
great, let's go with _B_ using the name __view__, so +1 for _D_ ;)